README.md

```diff
@@ -21,21 +21,16 @@ Install
 
 Requires [PHP CodeSniffer](http://pear.php.net/package/PHP_CodeSniffer/) version 3.x with PHP 5.4 or higher.
 
-Because of the way PHP CodeSniffer works, you need to put the `Security/` folder from phpcs-security-audit in `/usr/share/php/PHP/CodeSniffer/Standards` or do a symlink to it.
-
-The easiest way to install is to git clone and use composer that will create the symlink for you:
-```
-composer install
-./vendor/bin/phpcs --standard=example_base_ruleset.xml tests.php
-```
-
-The package is also on [Packagist](https://packagist.org/packages/pheromone/phpcs-security-audit):
+The easiest way to install is using [Composer](https://getcomposer.org/):
 ```
-composer require pheromone/phpcs-security-audit
-sh vendor/pheromone/phpcs-security-audit/symlink.sh
+composer require --dev pheromone/phpcs-security-audit
 ./vendor/bin/phpcs --standard=./vendor/pheromone/phpcs-security-audit/example_base_ruleset.xml ./vendor/pheromone/phpcs-security-audit/tests.php
 ```
 
+This will also install the [DealerDirect Composer PHPCS plugin](https://github.com/Dealerdirect/phpcodesniffer-composer-installer/) which will register the `Security` standard with PHP_CodeSniffer.
+
+It is also possible to install this based on a git clone. In that case, you will need to [register the package with PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-installed-standard-paths) yourself.
```

Comment:
this is probably just for me (or other lazy contributors), but I would like to have the steps on how to install from a git clone directly in the readme like it was before.

Reply:
Happy to bring them back in improved form, however, I wonder if that wouldn't be better placed in a `CONTRIBUTING.md` file rather than in the readme ?

Reply:
Probably, but for now let's keep it simple. Note also that I have 10 years of using the tool and other people have done as such, so I want to keep the documentation (and the way it is) without breaking them / preventing them to use the tool -- even if it's not the current best practices.

Back the the current line, I don't really need to do `phpcs --config-set installed_paths` if I just `composer install` in the cloned folder. It just works because the default is to point it to the current directly. So I have:
```
git clone
composer install
./vendor/bin/phpcs --config-set installed_paths ../../..
```

seems good?

Reply:
haha that's literally the same damn thing as before... but I'm afraid I have it wrong because when I first tried it it didn't work for me.. perhaps I had composer.lock and vendor/ already there when I did.